test/framework/cluster.go

```diff
@@ -675,6 +676,9 @@ func (e *ClusterE2ETest) Run(name string, args ...string) {
 		}
 
 		if errorMessage != "" {
+			if features.IsActive(features.CheckpointEnabled()) {
```

Comment:
could we make it some kind of option? This is mixing eks-a feature toggles with the framework and it stays a bit hidden. I would prefer it we could make it more explicit, where when we execute the command from the test we specify we expect that command to fail.

Reply:
Good point. Added it as a ClusterE2ETestOpt now. Thoughts?

Reply:
Had a few conversations offline on potentially using `CommandOpt` here instead. Ultimately decided to keep this as a `ClusterE2ETestOpt` since the CommandOpts are used for changes to the command itself or the binary that we use to run that command. Whether or not a failure is expected does not change those factors, so it seems to make more sense as is. Merging it like this for now to get test runs in with the feature, but we can do a fast follow-up as we see fit to decide on the best route to take for this.

cc @vivek-koppuru @g-gaston